pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -691,9 +709,25 @@ a job has finished, either successfully or unsuccessfully.
 Please notice that a pipeline *will not fail* if one or more commands in the
 `epilogue` fail to execute for some reason.
 
+There are three types of epilogue commands:
+
+1. Epilogue commands that are always executed. Defined with `always` in the
+   epilogue section.
+
+2. Epilogue commands that are executed when the job fails. Defined with
+  `on_pass` in the epilogue section.
+
+3. Epilogue commands that are executed when the job passes. Defined with
+   `on_fail` in the epilogue sections.
```

Comment:
`on_fail` is executed when job fails, and `on_pass` is executed when job passes :twisted_rightwards_arrows:

Reply:
fixed